tests/api/test_product.py

```diff
@@ -910,7 +878,6 @@ def test_create_product(
         $name: String!,
         $basePrice: Decimal!,
         $sku: String,
-        $quantity: Int,
```

Comment:
IMHO we shouldn't remove the tests for deprecated part of code. We will drop it at the same time as deprecated fields

Reply:
Hmm, I have a question though - this is the mutation that creates a product with a variant, but right now variant doesn't have `quantity` field. It's easier to keep fields as deprecated in queries, but in mutation?